scripts/ngi_reports

```diff
@@ -66,6 +66,8 @@ def make_reports (report_type, ngi_node='unknown', working_dir=os.getcwd(), pand
     output_mds = report.parse_template(template)
     for output_bn, output_md in output_mds.iteritems():
         try:
+            ####### Temp fix, need a clean solution ########
```

Comment:
The line has been added by you though? Did you just copy it from my hack elsewhere?